packs/data/feats.db/perfect-resistance.json

```diff
@@ -25,7 +25,35 @@
                 }
             ]
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.houseOfPerfection.element",
+                "priority": 1,
+                "value": ""
+            },
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.HouseOfPerfection.PerfectResistance.Label",
+                "option": "used-sop-ki-spell",
+                "toggleable": true
+            },
```

Comment:
Since this effect lasts for a specific time it should be an effect

Reply:
Fair enough. Was just trying to reduce the number of effects. I will make the change.